Gemfile

```diff
@@ -2,16 +2,16 @@ source 'https://rubygems.org'
 
 ruby RUBY_VERSION # Needed to consider & solve for Ruby version requirements
 
-gem 'berkshelf'
+gem 'berkshelf', '~> 4.3.5'
 gem 'chef', '~> 12.0'
-gem 'chefspec'
+gem 'chefspec', '~> 7.1'
 gem 'codeclimate-test-reporter', '~> 1.0.0'
-gem 'foodcritic'
-gem 'oneview-sdk', '~> 4.5.1'
+gem 'foodcritic', '~> 11.3'
```

Comment:
This is causing an error in the CI for Ruby 2.1:
```
    foodcritic (~> 11.3) was resolved to 11.3.0, which depends on
      ruby (>= 2.2.2)
```
Should likely add it in a condition, like the guard gem, and to use the latest version in Ruby 2.2+ and use the latest possible to Ruby 2.1.